src/sc2api/sc2_client.cc

```diff
@@ -521,8 +521,8 @@ bool ObservationImp::UpdateObservation() {
         return false;
     }
 
-    uint32_t next_game_loop = observation_->game_loop();
-    bool is_new_frame = next_game_loop != current_game_loop_;
+    const uint32_t next_game_loop = observation_->game_loop();
```

Comment:
Style: no need in marking `bool` and `int` as const, those are copies so changes in it don't matter.
Here and in other places.

Reply:
I believe these are done